Aftonbladet.js

```diff
@@ -0,0 +1,209 @@
+{
+	"translatorID": "235c5c47-0c04-4d77-9afe-e22265a017a9",
+	"label": "Aftonbladet",
+	"creator": "Jonatan Svensson Glad",
+	"target": "^https?://ww*\.aftonbladet\.se",
```

Comment:
>I suggest to do write in Scaffold: [...] which will result here into
Umm..I have had to copy that from Scaffold manually, is there a way to automaticly export all code from Scaffold here?